src/ui/pages/VerifySeedPhrase/VerifySeedPhrase.tsx

```diff
@@ -86,6 +86,9 @@ const VerifySeedPhrase = () => {
     );
     await SecureStorage.set(KeyStoreKeys.IDENTITY_ENTROPY, entropy);
 
+    await SecureStorage.set(KeyStoreKeys.SIGNIFY_BRAN, seedPhraseStore.bran);
+    dispatch(setCurrentOperation(OperationType.REINIT_APP));
```

Comment:
We don't need this new operation type. We aren't supposed to re-start the app when we've verified the seed phrase because we still haven't finished onboarding.

`initApp` should be called once on app start always.
- If onboarding complete, it starts like before.
- If onboarding not complete:
  - Seed phrase is verified and stored, then
  - SSI agent screen is done which boots and connects to KERIA
  - Done